tile_engine/ops/gemm/gemm_host_api.hpp

```diff
@@ -55,14 +55,14 @@ struct DataTypeTraits<ck_tile::pk_int4_t>
 };
 
 /**
- * @brief  trait for GEMM kernel
- * @param pipeline:   pipeline name
- * @param scheduler:  scheduler name
- * @param epilogue:  epilogue name
- * @param kPadM:     padding for M dimension
- * @param kPadN:     padding for N dimension
- * @param kPadK:     padding for K dimension
- *
+ * @brief  Defines the configuration parameters for a GEMM operation, enabling the selection of a
+ * specific kernel instance based on the provided settings.
+ * @param pipeline [std::string] The name of the pipeline.
+ * @param scheduler [std::string] The name of the scheduler (e.g., "intrawave", "interwave").
+ * @param epilogue [std::string] The name of the epilogue (e.g., "cshuffle", "default").
+ * @param kPadM [bool] Indicates whether padding is applied to the M dimension.
+ * @param kPadN [bool] Indicates whether padding is applied to the N dimension.
+ * @param kPadK [bool] Indicates whether padding is applied to the K dimension.
```

Comment:
Documenting argument types is a duplication (you can just look at the code a few lines below).

```suggestion
 * @param pipeline The name of the pipeline.
 * @param scheduler The name of the scheduler (e.g., "intrawave", "interwave").
 * @param epilogue The name of the epilogue (e.g., "cshuffle", "default").
 * @param kPadM Indicates whether padding is applied to the M dimension.
 * @param kPadN Indicates whether padding is applied to the N dimension.
 * @param kPadK Indicates whether padding is applied to the K dimension.
```

Reply:
Indeed the members doc should be above each member.